src/Nuget/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.Nuspec

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.AppDesigner</id>
+    <summary>AppDesigner designer host implementation for Visual Studio project property pages</summary>
+    <description>
+      This package implements the AppDesigner, which is the designer host used for project property pages among other things in Visual Studio
+    </description>
+    <language>en-US</language>
+    <version>15.0.0-alpha</version> <!-- This value will be overriden by the Nuget pack call. Having this value to satisfy the schema. -->
+    <authors>dotnet</authors>
+    <projectUrl>https://github.com/dotnet/roslyn-project-system/</projectUrl>
+    <dependencies>
+      <dependency id="Microsoft.VisualStudio.Designer.Interfaces" version="1.1.4322" />
+      <dependency id="Microsoft.VisualStudio.ManagedInterfaces" version="8.0.50727" />
+      <dependency id="Microsoft.VisualStudio.TextManager.Interop.10.0" version="10.0.30319" />
+      <dependency id="Microsoft.VisualStudio.Shell.Interop.10.0" version="10.0.30319" />
+      <dependency id="Microsoft.VisualStudio.Shell.Interop.11.0" version="11.0.61030" />
+      <dependency id="Microsoft.VisualStudio.TextManager.Interop.12.0" version="12.0.30110" />
+      <dependency id="Microsoft.VisualStudio.Shell.Interop.12.1.DesignTime" version="12.1.30328" />
+      <dependency id="Microsoft.VisualStudio.TextManager.Interop.12.1.DesignTime" version="12.1.30328" />
+      <dependency id="Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime" version="14.1.24720" />
+      <dependency id="Microsoft.VisualStudio.SDK.EmbedInteropTypes" version="14.1.2" />
+      <dependency id="Microsoft.VisualStudio.Shell.Design" version="15.0.25123-Dev15Preview" />
+      <dependency id="Microsoft.VisualStudio.Shell.15.0" version="15.0.25123-Dev15Preview" />
+      <dependency id="Microsoft.VisualStudio.Telemetry" version="15.0.691-master31907920" exclude="build"/>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="Microsoft.VisualStudio.AppDesigner.dll" target="lib" />
```

Comment:
This currentlt says "this library applies to every .NET platform ever shipped", that is not correct. The folder should be lib\net46 or whatever our minimum .NET Framework version is.